daydreams-agent/gasroute-oracle/.env

```
@@ -0,0 +1,3 @@
+AGENT_NAME=GasRoute Oracle
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)